### ISSUE TITLE
The "PyPy" CI job fails

### ISSUE THREAD

**Issue by vu3rdd** [LeastAuthority/magic-folder#66]
The job fails with the message below. Possible fix is to upgrade the openssl version?

```
pypy27-coverage inst: /tmp/magic-folder.tox/.tmp/package/1/magic_folder-0.0.0.post288.zip
ERROR: invocation failed (exit code 1), logfile: /tmp/magic-folder.tox/pypy27-coverage/log/pypy27-coverage-3.log
================================== log start ===================================
Traceback (most recent call last):
  File "/usr/local/lib-python/2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/local/lib-python/2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/tmp/magic-folder.tox/pypy27-coverage/site-packages/pip/__main__.py", line 16, in <module>
    from pip._internal import main as _main  # isort:skip # noqa
  File "/tmp/magic-folder.tox/pypy27-coverage/site-packages/pip/_internal/__init__.py", line 40, in <module>
    from pip._internal.cli.autocompletion import autocomplete
  File "/tmp/magic-folder.tox/pypy27-coverage/site-packages/pip/_internal/cli/autocompletion.py", line 8, in <module>
    from pip._internal.cli.main_parser import create_main_parser
  File "/tmp/magic-folder.tox/pypy27-coverage/site-packages/pip/_internal/cli/main_parser.py", line 12, in <module>
    from pip._internal.commands import (
  File "/tmp/magic-folder.tox/pypy27-coverage/site-packages/pip/_internal/commands/__init__.py", line 6, in <module>
    from pip._internal.commands.completion import CompletionCommand
  File "/tmp/magic-folder.tox/pypy27-coverage/site-packages/pip/_internal/commands/completion.py", line 6, in <module>
    from pip._internal.cli.base_command import Command
  File "/tmp/magic-folder.tox/pypy27-coverage/site-packages/pip/_internal/cli/base_command.py", line 20, in <module>
    from pip._internal.download import PipSession
  File "/tmp/magic-folder.tox/pypy27-coverage/site-packages/pip/_internal/download.py", line 15, in <module>
    from pip._vendor import requests, six, urllib3
  File "/tmp/magic-folder.tox/pypy27-coverage/site-packages/pip/_vendor/requests/__init__.py", line 97, in <module>
    from pip._vendor.urllib3.contrib import pyopenssl
  File "/tmp/magic-folder.tox/pypy27-coverage/site-packages/pip/_vendor/urllib3/contrib/pyopenssl.py", line 46, in <module>
    import OpenSSL.SSL
  File "/tmp/magic-folder.tox/pypy27-coverage/site-packages/OpenSSL/__init__.py", line 8, in <module>
    from OpenSSL import crypto, SSL
  File "/tmp/magic-folder.tox/pypy27-coverage/site-packages/OpenSSL/crypto.py", line 15, in <module>
    from OpenSSL._util import (
  File "/tmp/magic-folder.tox/pypy27-coverage/site-packages/OpenSSL/_util.py", line 6, in <module>
    from cryptography.hazmat.bindings.openssl.binding import Binding
  File "/tmp/magic-folder.tox/pypy27-coverage/site-packages/cryptography/hazmat/bindings/openssl/binding.py", line 205, in <module>
    _verify_openssl_version(Binding.lib)
  File "/tmp/magic-folder.tox/pypy27-coverage/site-packages/cryptography/hazmat/bindings/openssl/binding.py", line 169, in _verify_openssl_version
    "You are linking against OpenSSL 1.0.1, which is no longer "
RuntimeError: You are linking against OpenSSL 1.0.1, which is no longer supported by the OpenSSL project. You need to upgrade to a newer version of OpenSSL.
```

**Comment by vu3rdd** [comment]
Unfortunately, `pypy` official docker image for PyPy 2.7.x is Debian Jessie based which ships with OpenSSL 1.0.1. Jessie is very old at this point. Filed an issue at the upstream. https://github.com/docker-library/pypy/issues/36

**Comment by vu3rdd** [comment]
Fixed by #78.
